modules/exploits/multi/http/clipbucket_fileupload_exec.rb

```diff
@@ -0,0 +1,140 @@
+##
+# This module requires Metasploit: https://metasploit.com/download Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "ClipBucket Unauthenticated Arbitrary File Upload lower than 4.0.0 - Release 4902",
+      'Description'    => %q{
+        This module exploits a vulnerability found in ClipBucket version  Lower than 4.0.0 - Release 4902.
```

Comment:
Consider `before`.

`This module exploits a vulnerability found in ClipBucket versions before 4.0.0 (Release 4902).`